src/components/ebay-notice/marko-tag.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I believe this file (`marko-tag.json`) and the `marko.json` won't work in Marko v4, and that configuration needs to be moved up into the root `marko.json` of the project, per #15.

Therefore, please move this configuration up into the root `./marko.json`.

Reply:
Oh I missed it, my bad. Will correct it.